src/common/components/nav/subNav.tpl.html

```diff
@@ -30,7 +30,7 @@
     </ul>
 
     <div class="cart">
-      <div uib-dropdown on-toggle="$ctrl.cartOpened()" is-open="$ctrl.cartOpen" auto-close="outsideClick">
+      <div uib-dropdown on-toggle="$ctrl.cartOpened()" is-open="$ctrl.cartOpenSubNav" auto-close="outsideClick">
```

Comment:
Could you still open both cart dropdowns and use `on-toggle` to control the `is-open` value of the second one? Basically have the secondary one mirror the primary one. Idk if that would solve all the problems but it seems like that would keep you from having to figure out which one is currently opened.

I was looking at the code and it seems like having 2 `is-open`s probably toggles it twice. At least it seems like something is calling a close function and it deregisters the outsideClick handler.

Reply:
I tried that, but `$ctrl.cartOpened()` is fired causing a loop.

Reply:
Could you bypass the emit event call? Or only call that on the second one? Idk...